src/test/java/org/opensearch/ad/TestHelpers.java

```diff
@@ -1550,9 +1549,9 @@ public static DetectorValidationIssue randomDetectorValidationIssueWithDetectorI
     }
 
     public static ClusterState createClusterState() {
-        ImmutableOpenMap<String, IndexMetadata> immutableOpenMap = ImmutableOpenMap
-            .<String, IndexMetadata>builder()
-            .fPut(
+        Map<String, IndexMetadata> immutableOpenMap = new HashMap<>();
```

Comment:
I see the change is coming from OpenSearch but we shouldn't call the variable name as `immutableOpenMap` anymore. How about indexMap or if you have a better name?